cerbero/packages/packagesstore.py

```diff
@@ -166,6 +166,19 @@ def _load_packages(self):
         # Add packages by ascending pripority
         for key in sorted(packages.keys()):
             self._packages.update(packages[key])
+        # Add a package for every recipe
+        for recipe in self.cookbook.get_recipes_list():
+            p = self._package_from_recipe(recipe)
+            self._packages[p.name] = p
+
+    def _package_from_recipe(self, recipe):
+        p = package.Package(self._config, self, self.cookbook)
+        p.name = recipe.name
+        p.license = recipe.licenses
+        p.version = '%s-%s' % (recipe.version, self.cookbook.recipe_file_hash(recipe.name))
+        p.files = ['%s:libs' % p.name]
```

Comment:
I would do it differently, the recipes hace their own functions to list files, and we want packages to contain the same files as the recipe, otherwise extracting a binary would install an uncomplete list of files.
I would create a new RecipePackage that takes a recipe in the contstructor
and override list_files and list_devel_files with recipe.dist_files_list () and recipe.devel_files_list()